src/nfts/NftGallery.tsx

```diff
@@ -80,7 +81,12 @@ export default function NftGallery() {
             >
               <Touchable
                 borderless={false}
-                onPress={() => navigate(Screens.NftsInfoCarousel, { nfts: [item] })}
+                onPress={() =>
+                  navigate(Screens.NftsInfoCarousel, {
+                    nfts: [item],
+                    networkId: networkConfig.defaultNetworkId, // TODO(satish): pass the correct network id once we fetch nfts for all networks
```

Comment:
Is there an associated ticket for this todo?

Reply:
same issue, will be a follow up PR